src/main/java/teammates/common/datatransfer/TeamEvalResult.java

```diff
@@ -94,18 +94,22 @@ public TeamEvalResult(int[][] submissionValues) {
     }
 
     /**
-     * Replaces all missing points (for various reasons such as 'not sure' or
-     * 'did not submit') with NA.
+     * Replaces all missing points (for various reasons such as 'not sure' with NSU and
+     * 'did not submit' with NA).
```

Comment:
This sentence does not really flow well. You could either:
- From the original version, remove `with NA`.
- From the modified version, remove `for various reasons such as`.

Reply:
Done.